docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "2.1"
+
+services:
+
+  osqa:
+    build:
+      context: .
+    environment:
+      OSQA_DB_USER: osqa
+      OSQA_DB_PASSWORD: osqa
+      OSQA_DB_NAME: osqa
+      OSQA_DB_HOST: postgres
+
+    ports:
+      - 8080:8080
+    depends_on:
+      postgres:
+        condition: service_healthy
+      memcached:
+        condition: service_started
+
+  postgres:
+    image: postgres:10
+    environment:
+      POSTGRES_USER: osqa
+      POSTGRES_PASSWORD: osqa
```

Comment:
See above...

Reply:
Only used for testing locally. `docker-compose.yml` will not be used in production.